Applications/Spire/Source/KeyBindings/TaskKeyBindingsTableModel.cpp

```diff
@@ -14,43 +14,44 @@ namespace {
   const auto COLUMN_COUNT = 9;
 
   QVariant to_variant(const any& value) {
-    if(value.type() == typeid(Quantity)) {
+    const auto& type = value.type();
+    if(type == typeid(Quantity)) {
       return QVariant::fromValue(any_cast<Quantity>(value));
-    } else if(value.type() == typeid(Region)) {
+    } else if(type == typeid(Region)) {
       return QVariant::fromValue(any_cast<Region>(value));
-    } else if(value.type() == typeid(OrderType)) {
+    } else if(type == typeid(OrderType)) {
       return QVariant::fromValue(any_cast<OrderType>(value));
-    } else if(value.type() == typeid(Security)) {
+    } else if(type == typeid(Security)) {
       return QVariant::fromValue(any_cast<Security>(value));
-    } else if(value.type() == typeid(Side)) {
+    } else if(type == typeid(Side)) {
       return QVariant::fromValue(any_cast<Side>(value));
-    } else if(value.type() == typeid(TimeInForce)) {
+    } else if(type == typeid(TimeInForce)) {
       return QVariant::fromValue(any_cast<TimeInForce>(value));
     }
     return QVariant();
   }
 
   bool is_same_value(const QVariant& value, const QModelIndex& index) {
+    const auto data = index.data();
```

Comment:
I may be missing some details on Spire's conventions, but normally we don't use `const` for stack variables.